manifests/config.pp

```diff
@@ -41,7 +41,7 @@
   foreman_proxy::settings_file { 'tftp': }
   foreman_proxy::settings_file { 'realm': }
 
-  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun {
+  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun or tagged('foreman_proxy::plugin::salt') {
```

Comment:
@stbenjam yeah, that's acceptable.  We can always improve the usability from foreman-installer by using a kafo hook or something to change the value if the salt plugin is enabled, perhaps.

I think that @ekohl's suggestion is better though for the default, non-Augeas path.  Using concat for the sudoers file, or simply using a second sudoers file in the salt class would be good.  For the Augeas case, we could either come up with a second Augeas resource, or perhaps use datacat.